src/SDK/Language/Flutter.php

```diff
@@ -4,6 +4,37 @@
 
 class Flutter extends Dart {
 
+    /**
+     * @param $type
+     * @return string
+     */
+    public function getTypeName($type)
+    {
+        switch ($type) {
+            case self::TYPE_INTEGER:
+                return 'int';
+            break;
+            case self::TYPE_STRING:
+                return 'String';
+            break;
+            case self::TYPE_FILE:
+                return 'FileInput';
+            break;
+            case self::TYPE_BOOLEAN:
+                return 'bool';
+            break;
+            case self::TYPE_ARRAY:
+            	return 'List';
+			case self::TYPE_OBJECT:
+				return 'Map';
```

Comment:
Indentation here as well :sweat_smile:

Reply:
The indentation is good locally, only messed up in Github don't know what's the difference. If I fix here, it will be bad later when merged.